docs/UserGuide.md

```diff
@@ -156,16 +156,65 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+ContacX data are saved in the hard disk automatically after any command that changes the data in JSON format at `[JAR file location]/data/contacx.json`.
 
+### Exporting the data: `exportcsv`
+
+Exports the current list as a CSV file that can be imported through ContacX or other contact software such as Google/Outlook
+
+Format: `exportcsv EXPORTTYPE`
+* Exports CSV as per the specified format.
+* If `EXPORTTYPE` is left empty, it will default to `EXPORTTYPE = 1` and export in custom ContacX format
```

Comment:
If it can be left empty, then it should be surrounded by square brackets.

Reply:
thanks for the catch, didn't notice it when doing